core/tests/protractor_utils/waitFor.js

```diff
@@ -117,6 +120,17 @@ var newTabToBeCreated = async function(errorMessage, urlToMatch) {
   }, DEFAULT_WAIT_TIME_MSECS, errorMessage);
 };
 
+/**
+ * Check if a file has been downloaded
+ */
+
+var fileToBeDownloaded = async function(filename) {
+  var name = DPATH + '/' + filename;
```

Comment:
Can you use `path.resolve` here to make sure this works across different operating systems?

Reply:
I've added it